content/en/getting_started/synthetics/_index.md

```diff
@@ -0,0 +1,198 @@
+## Overview
+
+Datadog Synthetics offers two different ways to monitor your applications: API tests to monitor the uptime of your API endpoints and browser tests to check key user journeys. Your tests can be run either from managed locations or from private locations. Synthetics helps you ensure uptime, identify regional issues, and make sure key web transactions can be performed on your application.
+
+{{< img src="synthetics/synthetics_home_page.png" alt="Synthetics home page" responsive="true">}}
+
+By unifying Synthetics with your metrics, traces, and logs, Datadog allows you to observe how all your systems are performing, as experienced by your users. The [Synthetics][1] homepage details all of this information to give you real-time status updates, response times, and uptimes.
+
+This guide shows you how to set up your first Synthetics test with Datadog. Follow the sections below to configure an API test, browser test, or private location for internal-facing apps or private URLs.
+
+## Prerequisites
+
+If you haven't already, create a [Datadog account][2].
+
+## Configure your first tests
+
+### Create a browser test
+
+[Browser tests][3] are scenarios executed by Datadog on your web applications. They run at configurable periodic intervals from multiple locations around the world, and from multiple devices. These checks verify both that your applications are up and responding to requests, and that any conditions defined in your scenarios are met.
+
+In this example, a browser test is configured to map a user's journey from adding an item to cart to successful checkout. If any step during the browser test fails, it throws an error that is recorded in Datadog as a **Test Result**.
+
+{{< img src="getting_started/synthetics/browser-test.png" alt="Browser test" responsive="true" style="width:90%;" >}}
+
+#### Configure your test
+
+1. In the Datadog application, hover over **UX Monitoring** in the left hand menu and select **Synthetics Test**. 
+2. In the top right corner, click the **New Test** button. 
+3. Select **Browser Test**.
+4. Define the configuration of your browser test:
+    - Add a URL. In this example the URL is https://www.shopist.io/.
+    - Name the test.
+    - Set tags to help organize tests.
+    - Choose devices and locations for testing. In this example, the test is only run on **Large Laptop** & on **English speaking countries**.
+    - Specify a test frequency.
+    - Set alert conditions to determine the circumstances under which you want a test to send a notification alert. To avoid being alerted on network blips that might happen on specific locations, this test is configured as:
+        `An alert is triggered if your test fails for 0 minutes from any 3 of 13 locations`
+        `Retry 1 time before location is marked as failed`
```

Comment:
```suggestion
    - Set alert conditions to determine the circumstances under which you want a test to send a notification alert. 
      - If you want to avoid being alerted on network blips that might happen on isolated locations, you can for instance set:
        `An alert is triggered if your test fails for 0 minutes from any 3 of 13 locations`
      - If you want to be sure that a test execution is only considered a failure after 2 failed test runs, specify how many times you want your test to be re-run before being actually considered a failure:
        `Retry 1 time before location is marked as failed`
```